install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -8,6 +8,25 @@
 {{- $defaultKeepDeprecatedProbes = "false" -}}
 {{- end -}}
 
+{{- /* Workaround so that we can set the minimal k8s version that we support */ -}}
+{{- $k8sVersion := .Capabilities.KubeVersion.Version -}}
+{{- $k8sMajor := .Capabilities.KubeVersion.Major -}}
+{{- $k8sMinor := .Capabilities.KubeVersion.Minor -}}
+
+{{- if .Values.Capabilities -}}
+{{- if .Values.Capabilities.KubeVersion -}}
+{{- if .Values.Capabilities.KubeVersion.Version -}}
+{{- $k8sVersion = .Values.Capabilities.KubeVersion.Version -}}
+{{- if .Values.Capabilities.KubeVersion.Major -}}
+{{- $k8sMajor = toString (.Values.Capabilities.KubeVersion.Major) -}}
+{{- if .Values.Capabilities.KubeVersion.Minor -}}
+{{- $k8sMinor = toString (.Values.Capabilities.KubeVersion.Minor) -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
```

Comment:
Instead of repeating this block in many files, can't we simply add this to a template file, say  `install/kubernetes/cilium/templates/_kubeversion.yaml `?

Reply:
I'll try to see if that works.

Reply:
I'm pretty sure this is possible, just not sure how... I and @aanm tried several things to no avail.

Reply:
@sayboras do you want to give it a shot?